tc_driver/Cases.jl

```diff
@@ -226,21 +266,26 @@ end
 function initialize_profiles(::Soares, grid::Grid, param_set, state; kwargs...)
     aux_gm = TC.center_aux_grid_mean(state)
     prog_gm = TC.center_prog_grid_mean(state)
-    ρ_c = prog_gm.ρ
     FT = TC.float_type(state)
     prof_q_tot = APL.Soares_q_tot(FT)
     prof_θ_liq_ice = APL.Soares_θ_liq_ice(FT)
     prof_u = APL.Soares_u(FT)
     prof_tke = APL.Soares_tke(FT)
+    # Solving the initial value problem
+    p_0::FT = FT(1000 * 100) # TODO - duplicated from surface_ref_state
```

Comment:
I honestly think that copying `p_0` is fine, the entire `Cases` structure will change at the end

Reply:
How about I put all the surface values in the `AtmosphericProfilesLibrary.jl`? This way I could load them in both places, without duplicating. Would that be ok @charleskawczynski?

Reply:
Never mind. Will just hard code them twice for now.